### ISSUE TITLE
Do you transform the images from BGR to YCbCr color space? 

### ISSUE THREAD

**Issue by majian-stu** [calmevtime/DCTNet#37]
Hi! Do you transform the images from BGR to YCbCr color space before converting them to the frequency domain?
As you have mentioned in your paper, “Then images are transformed to the YCbCr color space and converted to the frequency domain (DCT transform in Figure 2)."
But I cannot find any operation that transforms the image from BGR to YCbCr in your open-source code, and you feed the BGR images to the function "transform_dct" directly.
I'm not sure if I missed any details. I hope you can help me.

**Comment by jiaozizhao** [comment]
Same question. I found there is a parameter called 'backend' in class DatasetFolderDCT, which is used to control whether to load YCrCb image. But I noted the parameter is set to default as 'opencv', which means load BGR image. Then I can not find where BGR image is converted to YCrCb. 
Could you please give some details? Thanks very much.

**Comment by jiaozizhao** [comment]
Or did you forgot to set the parameter 'backend' as ''dct" in your dataloader file dataloader_imagenet_dct.py (for classification)

line 53-56:
    val_loader = torch.utils.data.DataLoader(
        ImageFolderDCT(valdir, transform),
        batch_size=args.test_batch, shuffle=False,
        num_workers=args.workers, pin_memory=True)

**Comment by J-JunChen** [comment]
You can find the DCT processing in the `TransformUpscaledDCT(object)` class in `cvtransforms.py` file.
